games-strategy/7kaa/7kaa-2.15.4p1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Seven Kingdoms: Ancient Adversaries"
+DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy\
+	with the addition of trade, diplomacy, and espionage.\
+	\
+	The game enables players to compete against up to six other kingdoms allowing players to conquer \
+	opponents by defeating them in war (with troops or machines), capturing their buildings with spies, \
+	or offering opponents money for their kingdom."
+HOMEPAGE="https://www.7kfans.com/"
+COPYRIGHT="Enlight Software Ltd."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/the3dfxdude/7kaa/releases/download/v2.15.4p1/7kaa-2.15.4p1.tar.xz"
+CHECKSUM_SHA256="f0265c077025d77b435f1c10b8ba178bdb16329e51ab735330ebaa71a104fbf8"
+PATCHES="7kaa-2.15.4p1.patchset"
+ADDITIONAL_FILES="7kaa.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	7kaa$secondaryArchSuffix
+	app:sevenKingdoms
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libenet$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:curl_config$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libsdl2$secondaryArchSuffix
+	devel:libenet$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD() {
+	# for some reason the configure script doesn't find these variables by itself
+	CURL_CFLAGS=$(curl-config --cflags)
+	CURL_LIBS=$(curl-config --libs)
+	export CURL_CFLAGS
+	export CURL_LIBS
```

Comment:
From github I see there is a configure.ac file there, maybe run `autoreconf -vfi runConfigure ...` ?

Reply:
The official release source has no autoconf files, though I could theoretically use the github repository

Reply:
OK, haven't checked a build yet, will report back later :)